uktrade/directory-cms
# Improve international article page video/image functionality

This PR makes two changes to the `article_video` and `article_image` fields on `InternationalArticlePage` pages:
1. It takes the video and image out of the 'images' panel, and adds them to the general content panels, to be easier located by users. This change required the fields to be added to the translatable fields for that model, and comes with a migration.
2. Updates the serializer used for the `article_video` field to use the newer `GreatMediaSerializer`, which also serialises a video's transcript and subtitles.

**Screenshot - wagtail panels new**
![Screenshot 2023-07-04 at 16 15 06](https://github.com/uktrade/directory-cms/assets/22460823/4c3a019a-f7c0-41d8-a18c-e969aab33935)

To test locally:
- Assuming you have a copy from the directory-cms dev db, go to http://cms.trade.great:8010/admin/pages/829/edit/
- The article video and image fields should appear below the other article fields in the main content edit section. There should no longer be an 'Images' panel at the top.

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-811

### Reviewing help


### Merging
